contracts/token/ERC20/ERC20.sol

```diff
@@ -133,9 +133,6 @@ abstract contract ERC20 is Context, IERC20, IERC20Metadata, IERC20Errors {
     /**
      * @dev See {IERC20-transferFrom}.
      *
-     * Emits an {Approval} event indicating the updated allowance. This is not
-     * required by the ERC. See the note at the beginning of {ERC20}.
-     *
```

Comment:
In my opinion, this section is important because it makes it explicit that the ERC does not require the event. That's still true. Let's rephrase it to: 

```suggestion
Skips emitting an {Approval} event indicating an allowance update. This is not
required by the ERC. See {xref-ERC20-_approve-address-address-uint256-bool-}.
```